index.html

```diff
@@ -0,0 +1,198 @@
+      <nav class="smallMenu">
+        <ul class="darkMenu">
+          <li><a href="#"><img class="facebook" src="./imgs/facebook-logo.png" alt="icone"/></a></li>
+          <li><a href="#"><img class="facebook" src="./imgs/twitter.png" alt="icone"/></a></li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Profile</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </nav>
+      <nav class="bigMenu">
+        <button class="hamburger" type="button">
+          <span class="menu icon" id="icon">
+            <img onclick="openMenu()" class="showMenu" src="./imgs/menu.png" alt="menu">
+          </span>
+         </button>
+        <ul class="nav-element">
+         <li><a class="logo" href="#"><img src="assets/img/coincity.png" alt="Logo image"/></a></li> 
+          <li><a  href="index.html">Home</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li><a class="webinar" href="#">Webinar</a></li>
+          <li><img onclick="closeMenu()" class="showMenu close_btn" src="./imgs/close.png" alt=""></li>
+        </ul>
+      </nav>
+      <nav class="desktop_navbar">
+        <img  class="header_logo" src="./imgs/brain.png" alt="brain">
+        <ul class="navbar_lst">
+           <li><a href="index.html">Home</a></li>
+           <li><a href="#">Program</a></li>
+           <li><a href="about.html">About</a></li>
+           <li><a href="#">Sponsor</a></li>
+           <li><a href="#">News</a></li>
+         </ul>
+         <a class="webinar_btn" href="#">Webinar</a>
+      </nav>
+    </header>
+    <section class="containerOne">
+      <div class="imageDiv">
+        <h3>"Hello!Sharing World"</h3>
+        <h1>Creative commons <br> Summit<br>2015</h1>
+        <p>A joyful celebration believing in the value of openness and sharing, creating a postive  change with people from over 80 countries is taking place in october, in Korea.
+        </p>
+        <time datetime="2022-01-11T13:53">2015.10.15(THU) ~ 16(FRI)</time>
+        <h5>@ National Museum of Korea, Art Centre Nabi and more</h5>
+      </div>
+    </section>
+    <section class="programs">
+      <div class="center-container">
+        <h2>Main Program</h2>
+        <hr class="programLine" />
+      </div>
+      <div class="progContainer">
+        <div class="programCards">
+          <i class="fa fa-users fa-2x"></i>
+          <div><h3>Lecture</h3></div>
+          <p>
+            Many national governments around the world are showing themselves to
+            be inefficient
+          </p>
+        </div>
+        <div class="programCards">
+          <i class="fa fa-rocket fa-2x"></i>
+          <div><h3>Exhibition</h3></div>
+          <p>
+            Many national governments around the world are showing themselves to
+            be inefficient
+          </p>
+        </div>
```

Comment:
- [ ] Thank you for adjusting the CSS for the main program section. please notice that now the text is breaking outside the container. Please adjust the width of each container to ensure that this doesn't happen. 
## Your Design notice the breaking. 
![image](https://user-images.githubusercontent.com/18730106/147385665-01c924ef-df2c-43c3-9cda-7b18ab386fb6.png)